### ISSUE TITLE
@click event doesnot work

### ISSUE THREAD

**Issue by IakobMirotadze** [nuxt/nuxt.js#13907]

**Comment by yuvraj339** [comment]
Just do one thing, run below command will fix issue at my end 
**npm run build**, when build successfully run below command again
**npm run dev**

**Comment by huntz20** [comment]
> Just do one thing, below commands fixed issues at my end **npm run build**, when build successfully run below command again **npm run dev**

weirdly is also fix for me with this approach

**Comment by AntMam** [comment]
Hey everyone, seems like this is still an issue. I made a simple reproduction here:
https://stackblitz.com/edit/github-jsuxnz-rst5gg?file=README.md,nuxt.config.ts,app.vue

When runtimeConfig is used, @click event doesn't work anymore. If you remove runtimeConfig variables, everything starts to work again. Has anyone figured this out?

**Comment by leonwetterich** [comment]
Unfortunately still seems to be an issue

**Comment by Proffett** [comment]
i still have the same

**Comment by 60late** [comment]
Same problem. 
In my case, i set `ssr: true` in `nuxt.config.ts`,and every click event is not working.
If i set `ssr: false`, everything just works fine.
Why?????

**Comment by danielroe** [comment]
@60late It sounds like you have a hydration error. You can install `@nuxtjs/html-validator` which can help resolve if the issue is invalid HTML. Check out your console in development to see the logs.

**Comment by 60late** [comment]
> @60late It sounds like you have a hydration error. You can install `@nuxtjs/html-validator` which can help resolve if the issue is invalid HTML. Check out your console in development to see the logs.
Thanks.
I checked my code, finally find out there's something wrong with my data fetching process.
I fixed it, now it's working just fine.

**Comment by kacperknapik** [comment]
For anyone looking for help. What helped me was [this stackoverflow answer](https://stackoverflow.com/a/75285020/5132836). 

TL;DR - Run `build` command (eg. `npm run build`) to find any issues that are not visible right away in the console.

Not necessarily solving the `@click` issue per se, but helps a bit with debugging issues outside of it.
